traffic-advice.md

```diff
@@ -0,0 +1,32 @@
+# Traffic Advice
+
+Publishers may wish not to accept traffic from [private prefetch proxies](README.md) and other sources other than direct user traffic, for instance to reduce server load due to speculative prefetch activity.
+
+We propose a well-known "traffic advice" resource, analogous to `/robots.txt` (for web crawlers), which allows an HTTP server to declare that implementing agents should stop sending traffic to it for some time.
+
+## Proposal
+
+Agents which respect traffic advice should fetch the well-known path `/.well-known/traffic-advice.json`. If it returns a `200 OK` response with the `application/json` MIME type, the response body should contain valid JSON like the following:
```

Comment:
This would be the first web spec that uses file extensions from what I understand. I'd really strongly recommend we not do this. That means you do need server configuration access, but you also should need server configuration access to modify /.well-known anyway.

Reply:
Okay. I suspect modifying response headers is more work than adding a file in the web root, but probably at the same privilege level and hopefully not too difficult.